taggit/managers.py

```diff
@@ -69,7 +69,9 @@ def __init__(self, through, model, instance, prefetch_cache_name, ordering=None)
         if ordering:
```

Comment:
if we're going to set a default for `self.ordering`, let's make it so that we can still pass in `ordering=[]`, by  changing this `if ordering` to `if ordering is not None`.

That way in the changelog we can offer people to have the old behavior by passing that in explicitly